src/main.ts

```diff
@@ -49,6 +49,9 @@ export async function run() {
       auth.configAuthentication(registryUrl, alwaysAuth);
     }
 
+    const corepack = core.getInput('corepack') || 'false';
```

Comment:
In order the input to be recognized it must be defined in [action.yml](https://github.com/actions/setup-node/blob/main/action.yml)